plugins/rapid7_insightvm_cloud/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: rapid7_insightvm_cloud
 title: Rapid7 InsightVM Cloud
 description: InsightVM is a powerful vulnerability management tool which finds, prioritizes, and remediates vulnerabilities. This plugin uses the InsightVM Cloud Integrations API to view assets and start scans
-version: 2.2.0
+version: 2.2.1
```

Comment:
```suggestion
version: 3.0.0
```
This will be a major version increment to 3.0.0 due to backwards-incompatible changes (removing properties from output types). Make sure to regenerate the plugin after updating the version here.